tests/test_numseq.py

```diff
@@ -2,24 +2,12 @@
 # -*- coding: utf-8 -*-
 
 import unittest
-# import subprocess
 import sys
 import StringIO
 
 
 class TestNumSeq(unittest.TestCase):
-    # def test_help(self):
-    #     """ Running the program without arguments should show usage. """
-
-    #     # Run the command `python ./echo.py -h` in a separate process, then
-    #     # collect it's output.
-    #     process = subprocess.Popen(
-    #         ["python", "./echo.py", "-h"],
-    #         stdout=subprocess.PIPE)
-    #     stdout, _ = process.communicate()
-    #     usage = open("./USAGE", "r").read()
-
-    #     self.assertEqual(stdout, usage)
+    """test cases defined in assignment to check modules"""
 
     def test_fib(self):
         """make sure fib prints expected fibonacci sequence"""
```

Comment:
The summary line of docstrings should be capitalized and end with a period.
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Reply:
Thanks for the info. have updated the docstrings